src/Workspaces/Core/Portable/Microsoft.CodeAnalysis.Workspaces.csproj

```diff
@@ -27,6 +27,7 @@
   <ItemGroup Label="Linked Files">
     <Compile Remove="Storage\Sqlite\**\*.cs" Condition="'$(DotNetBuildFromSource)' == 'true'" />
     <Compile Include="..\..\..\CodeStyle\Core\CodeFixes\FixAllContextHelper.cs" Link="CodeFixes\FixAllOccurrences\FixAllContextHelper.cs" />
+    <Compile Include="..\..\..\Compilers\Core\Portable\InternalUtilities\SharedStopwatch.cs" Link="InternalUtilities\SharedStopwatch.cs" />
```

Comment:
><Compile Include="..\..\..\Compilers\Core\Portable\InternalUtilities\SharedStopwatch.cs" Link="InternalUtilities\SharedStopwatch.cs" /> [](start = 4, length = 135)

It looks like ```AnalyzerExecutor``` is the only consumer of the new type and it is not linked explicitly by this project. Are there other consumers I missed?

Reply:
There are no current consumers, but I am planning to replace some other uses of `Stopwatch` with this type for improved performance.

Reply:
> There are no current consumers, but I am planning to replace some other uses of Stopwatch with this type for improved performance.

I think the project file should be changed in the same PR that makes use of the new type in the project.

---
In reply to: [371411528](https://github.com/dotnet/roslyn/pull/41205#discussion_r371411528) [](ancestors = 371411528)

Reply:
Removed the unnecessary project change